manifests/pipelines/aiedge-e2e.pipeline.yaml

```
@@ -136,7 +136,22 @@ spec:
     - input: "$(params.fetch-model)"
       operator: in
       values: ["git"]
-  # Use kserve-download-model container image to fetch from S3 storage
+  - name: move-model-to-root-dir
+    params:
+    - name: model-name
+      value: $(params.model-name)
+    - name: subdirectory
+      value: model_dir-$(params.model-name)
+    - name: src-model-relative-path
+      value: $()
```

I think you forgot to fill this with the correct param?
```suggestion
      value: $(params.model-relative-path)
```
BTW these suggestions you can just accept I think and they will be committed directly.